moto/kinesis/responses.py

```diff
@@ -24,7 +25,9 @@ def is_firehose(self):
     def create_stream(self):
         stream_name = self.parameters.get('StreamName')
         shard_count = self.parameters.get('ShardCount')
-        self.kinesis_backend.create_stream(stream_name, shard_count, self.region)
+        stream = self.kinesis_backend.create_stream(stream_name, shard_count, self.region)
+        if isinstance(stream, BadRequest):
```

Comment:
Is this just the result of a bug in `create_stream`? I think `create_stream` should be raising the exception.